pkgs/firefox-nightly/default.nix

```
@@ -14,8 +14,10 @@
 let
   mach = buildMozillaMach rec {
     pname = "firefox-nightly";
+    binaryName = pname;
```

```suggestion
    binaryName = "firefox-nightly";
```

Use full value here. If someone overrides pname, this package will be good as broken.

---

https://github.com/NixOS/nixpkgs/commit/9df487993daabd4a7b126c87ec7e3b2d7be455a5
Should probably change it in nixpkgs as well as nixpkgs does the same thing.